IdentitySdkCore/IdentitySdkCore/Classes/ReachFivePassword.swift

```diff
@@ -2,12 +2,12 @@ import Foundation
 import BrightFutures
 
 public extension ReachFive {
-    func signup(profile: ProfileSignupRequest, scope: [String]? = nil) -> Future<AuthToken, ReachFiveError> {
+    func signup(profile: ProfileSignupRequest,redirectUrl: String? = nil, scope: [String]? = nil) -> Future<AuthToken, ReachFiveError> {
         let signupRequest = SignupRequest(
             clientId: sdkConfig.clientId,
             data: profile,
             scope: (scope ?? self.scope).joined(separator: " "),
-            acceptTos: nil
```

Comment:
This should have been done in a separate PR, it's completely unrelated to tests.

Reply:
This PR was based on an old PR (Add the redirectURL parameter in the signup method) (https://github.com/ReachFive/identity-ios-sdk/pull/74) which was merged